reporters/html_reporter.py

```diff
@@ -0,0 +1,25 @@
+from reporters.plain_reporter import PlainReporter
+from jinja2 import Environment
+
+
+class HTMLReporter(PlainReporter):
+    def __init__(self):
+        super().__init__()
+
+    # Override this method
+    def print_message_ids(self):
+        # Sort the messages.
+        self.sort_messages()
+
+        #template = Environment(keep_trailing_newline=True, autoescape=False).from_string(HTML)
+        template = Environment().get_template('template.txt')
+        f = open('output.html', 'w')
+
+        for msg in self._messages:
+            code = msg.msg_id
+            msg_new = msg.msg.replace('\n', '<br/>')
+            msg = msg._replace(msg=msg_new)
+            f = open('output.html', 'a')
+            f.write(template.render(code=code,symbol=msg.symbol,obj=msg.obj,msg=msg.msg))
+            print(template.render(code=code,symbol=msg.symbol,obj=msg.obj,msg=msg.msg))
```

Comment:
@david-yz-liu: I was thinking of creating a new issue for outputting to a file in general for all reporters. What do you think?